Cargo.toml

```
@@ -70,3 +71,7 @@ path = "./examples/datafusion_integration.rs"
 [[bin]]
 name = "orc-metadata"
 required-features = ["cli"]
+
+[[bin]]
+name = "orc-stats"
+required-features = ["cli"]
```

nit: Add an empty line
```suggestion
required-features = ["cli"]

```